modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc

```diff
@@ -0,0 +1,270 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-precache-feature_{context}"]
+= Using the container image pre-cache feature
+
+The SNO spoke clusters might have limited bandwidth to the container image registry, which can cause a timeout before the updates are completed. 
+The pre-cache feature allows the required container images to be present on the spoke cluster before the update maintenance time.
+
+[NOTE]
+====
+The maintenance window is not set by the Topology Aware Lifecycle Operator. You can apply the Topology Aware Lifecycle Operator CR at the beginning of the maintenance window by manual application or by external automation. The update procedure start when the `enable` field is set to `true`.
```

Comment:
For clarity does the "update procedure" include pre-caching or is pre-caching done before the CGU is enabled?

Reply:
@irinamihai Could you please check? My understanding here was that the pre-caching is done at this point. Once you set `enabled` to `true`, that's when updating the clusters starts.

Reply:
@vitus133 should have the last say here.
In my opinion, this should be rephrased to better explain precaching. The above paragraph is accurate, but it doesn't explain how pre-caching works.

Pre-caching is independent of the `enable` flag, as it has its own boolean flag, `preCaching`. If the `preCaching` field is set to true, then in the upgrade procedure we're firstly doing the precaching and once that is completed successfully, we are moving to the part where we're starting the policy remediation actions. These actions start when the `enable flag is set to `true`.

Reply:
@irinamihai is absolutely right here ^^

Reply:
Added a paragraph for clarification to the Using the pre-cache feature section:
> The container image pre-caching starts when the `preCaching` field is set to `true` in the `ClusterGroupUpgrade` CR. After a successful pre-caching process, you can start remediating policies. The remediation actions start when the `enable` field is set to `true`.